server/src/routes/api/v1/reviewRouter.js

```diff
@@ -0,0 +1,33 @@
+import express from "express"
+import objection from "objection"
+const { ValidationError } = objection
+
+import { Review } from "../../../models/index.js"
+import cleanUserInput from "../../../services/cleanUserInput.js"
+
+const reviewRouter = new express.Router()
+
+reviewRouter.post("/", async (req, res) => {
+  const { body } = req
+  const formInput = cleanUserInput(body)
+  const { comment, rating, trailId } = formInput
+
+  try {
+    const newReview = await Review.query().insertAndFetch({
+      comment,
+      rating,
+      trailId,
+      userId: 1,
```

Comment:
This will assign EVERY review to the first user! We want to make use of the current user. This is made available to us through `req.user`

Reply:
This is tempoary - figuring out how to pass down the currentUser state

Reply:
now updated